WcaOnRails/app/javascript/edit-events/EditEvents.jsx

```diff
@@ -101,6 +102,12 @@ function RoundsTable({ wcifEvents, wcifEvent }) {
             let roundFormatChanged = e => {
               let newFormat = e.target.value;
               wcifRound.format = newFormat;
+              if (wcifRound.cutoff) {
```

Comment:
I don't love how this will silently clear the cutoff if the user changes the round format and changes it back. How do you feel about prompting the user instead?

Philosophically, I think this is similar to the question of how to handle removing some rounds. We don't want the user to change the number of rounds from 4 to 2 and then back to 4, and be surprised that all their information for rounds 3 and 4 has disappeared. Instead, we prompt the user when they're changing the number of rounds from 4 -> 2 to make sure that this is what they want to do. Note that we don't prompt the user when they go from 2 -> 4. The idea is that we only prompt them if they're about to take an action that is not "trivially reversible".

Reply:
Sounds good to me, I changed the code accordingly!